src/main/java/mekanism/common/tile/component/SideConfig.java

```diff
@@ -0,0 +1,56 @@
+package mekanism.common.tile.component;
+
+import net.minecraft.util.EnumFacing;
+
+import java.util.Observable;
+import java.util.Observer;
+
+public class SideConfig {
+    private byte[] directions = new byte[EnumFacing.VALUES.length];
+    private Observable observable = new Observable();
```

Comment:
Hm, right again. I disagree with extending it personally. I don't disagree with this object having those set of methods, I just disagree with Observable being the superclass. But it seems that is actually the intent from the author of the class. "It can be subclassed to represent an object that the application wants to have observed." I get that the reason `setChanged` is protected is probably so that the observer itself is the only one to decide when it has changed, but I think my explanation from before is valid.

I think the easy thing is to just extend Observable. I think the correct thing is to either have an inner class which subclasses Observable and gives the ability to set changed to the SideConfig class; or to implement the observable pattern from scratch. I'd probably do the latter, personally.

Though at the moment, I'd probably sooner just delete the Observable from the class, since it's not being used.